utilities/io_tools/output_tools.py

```diff
@@ -85,8 +85,6 @@ def write_analysis_output(results, outfile, args, update_name=True):
     to_append = []
     if args.theoryCorr and not args.theoryCorrAltOnly:
         to_append.append(args.theoryCorr[0]+"Corr")
-    if args.maxFiles is not None:
-        to_append.append(f"maxFiles{args.maxFiles}")
```

Comment:
Can we keep this line, possibly removing it only when maxFiles is negative? It is useful to avoid overriding a file when you just run a test with --maxFiles 2. Perhaps one should also distinguish the case (dealt with in another PR) where -1 is all files while "< -1 or None" means 1:1 MC/data, but this is less important, since these two cases are not supposed to be used one after the other

Reply:
Ok added this back with updated string formatting.